app/models/concerns/comment/publishing.rb

```diff
@@ -24,7 +24,9 @@ def to_event_stream
         focus_id: focus_id,
         focus_type: focus_type,
         project_id: project_id,
+        category: category,
         section: section,
+        body: body,
```

Comment:
We're limited to 1MB per stream message, but so far our biggest comment is less than 10kb.